pdns/pdns_recursor.cc

```diff
@@ -3516,6 +3548,10 @@ int main(int argc, char **argv)
     ::arg().set("xpf-allow-from","XPF information is only processed from these subnets")="";
     ::arg().set("xpf-rr-code","XPF option code to use")="0";
 
+    ::arg().set("min-udp-source-port", "Minimum UDP port to bind on")="1025";
+    ::arg().set("max-udp-source-port", "Maximum UDP port to bind on")="65535";
+    ::arg().set("avoid-udp-source-port", "List of comma separated UDP port number to avoid")="11211";
```

Comment:
Small nit:

Perhaps name the settings `udp-source-port-$SETTING` so they are sorted nicely in the generated config file.

Reply:
this is kinda why i thought having a single udp-source-port setting would be a lot nicer and maybe easier to understand